adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -159,8 +165,41 @@ class Transform(protected val args: TransformArgs) extends BDGSparkCommand[Trans
         sc.loadFastq(args.inputPath, Option(args.pairedFastqFile), Option(args.fastqRecordGroup), stringency)
       } else if (args.forceLoadIFastq) {
         sc.loadInterleavedFastq(args.inputPath)
-      } else if (args.forceLoadParquet) {
-        sc.loadParquetAlignments(args.inputPath)
+      } else if (args.forceLoadParquet ||
+        args.limitProjection ||
+        args.useAlignedReadPredicate) {
```

Comment:
is it weird to have `args.limitProjection || args.useAlignedReadPredicate` cause interpretation of a file as parquet? would we rather throw an error if someone tries to load a `.bam` file with one of these flags? I'm not sure, just wondering